src/nspawn/nspawn.c

```diff
@@ -2378,7 +2299,10 @@ static int copy_devnodes(const char *dest, bool enable_fuse) {
                         return r;
         }
 
-        if (enable_fuse) {
+        /* Create /dev/fuse only when it is accessible. The check is necessary, as some custom service
+         * units that invoke nspawn may enable DevicePolicy= without DeviceAllow= for the device node. */
+        _cleanup_close_ int fuse_fd = open("/dev/fuse", O_CLOEXEC|O_RDWR);
```

Comment:
would prefer if this was hidden in some helper, and if we'd debug log if we see some unexpected error